src/http/ratelimiting.rs

```diff
@@ -209,6 +233,13 @@ impl Ratelimiter {
                         parse_header::<f64>(response.headers(), "retry-after")?
                     {
                         debug!("Ratelimited on route {:?} for {:?}s", route, retry_after);
+                        (self.ratelimit_callback)(RatelimitInfo {
+                            timeout: Duration::from_secs_f64(retry_after),
+                            limit: 1, // is this correct?
```

Comment:
`limit` specifies how many requests are allowed in a given period before the counter is reset, which for the [global rate limit is 50](https://discord.com/developers/docs/topics/rate-limits#global-rate-limit).
```suggestion
                            limit: 50,
```